examples/gallery/embellishments/map_roses.py

```diff
@@ -0,0 +1,148 @@
+r"""
+Directional map roses
+=========
+
+The ``rose`` parameter of the :meth:`pygmt.Figure.basemap` and
+:meth:`pygmt.Figure.coast` methods is used to add a directional map 
+rose to a map. This example shows how such a map rose can be customized:
+
+ - position: **g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**. Set the position
+   of the reference point. Choose from
+
+   - **g**: Give map coordinates as *longitude*\/\ *latitude*.
+   - **j**\|\ **J**: Specify a two-character (order independent) code.
+     Choose from vertical **T**\(op), **M**\(iddle), or **B**\(ottom) and
+     horizontal **L**\(eft), **C**\(entre), or **R**\(ight). Lower /
+     uppercase **j** / **J** mean inside / outside of the map bounding
+     box.
+   - **n**: Give normalized bounding box coordinates as *nx*\/\ *ny*.
+   - **x**: Give plot coordinates as *x*\/\ *y*.
+
+- width: **+w**: Set the width of the rose in plot coordinates (append
+  **i** (inch), **cm** (centimeters), or **p** (points).
+
+- fanciness: **+f**[level]: Get a “fancy” rose, and optionally specify the
+  level of fanciness. Level 1 draws the two principal E-W, N-S orientations,
+  2 adds the two intermediate NW-SE and NE-SW orientations, while 3 adds
+  the eight minor orientations WNW-ESE, NNW-SSE, NNE-SSW, and ENE-WSW
+  [default is 1].
+
+- justify: **+j**: Set the anchor point. Specify a two-character (order
+  independent) code. Choose from vertical **T**\(op), **M**\(iddle), or
+  **B**\(ottom) and horizontal **L**\(eft), **C**\(entre), or **R**\(ight).
+
+- label: **+l**[w,e,s,n]: Label the cardinal points W,E,S,N. Optionally,
+  append your own four comma-separated strings to override the default.
+  Skip a specific label by leaving it blank.
+
+- offset: **+o**\ *offset* or **+o**\ *xoffset*/\ *yoffset*: Give either a
+  common shift or individual shifts in x- (longitude) and y- (latitude)
+  directions.
+
+Colors of the map roses can be adjusted using :gmt-term:`MAP_DEFAULT_PEN`
+and :gmt-term:`MAP_TICK_PEN_PRIMARY` via :func:`pygmt.config`. Customizing
+label font and color can be done via :gmt-term:`FONT_TITLE`.
+"""
+
+fig = pygmt.Figure()
```

Comment:
Lost the `import pygmt` somehow, maybe during copying 🙃.

```suggestion
"""
import pygmt

fig = pygmt.Figure()
```